doc/api/esm.md

```diff
@@ -685,6 +685,12 @@ of Node.js applications.
 <!-- YAML
 added: v8.8.0
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/48842
+    description: Added `initialize` hook and deprecated `getGlobalPreload`.
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/48559
+    description: Allow loaders to use `import()` and unflag `Module.register`.
```

Comment:
Can you please move that to its own PR? Otherwise the `REPLACEME` will likely be incorrect depending on when we backport each PR.